source/configure/smtp-email.rst

```diff
@@ -6,126 +6,95 @@ SMTP email setup
 
-1. **Set up an SMTP email sending service** (if you don't yet have an SMTP service with credentials)
+.. tip::
 
-    * Any SMTP email service can be used, you just need the following
-      information: ``Server Name``, ``Port``, ``SMTP Username``, and
-      ``SMTP Password``.
+    If you don't have an SMTP service you can set one up with `Amazon Simple Email Service (SES) <https://aws.amazon.com/ses/>`_:
 
-    * If you don't have an SMTP service you can set one up with:
+    1. Go to `Amazon SES console <https://console.aws.amazon.com/ses>`__ then **SMTP Settings > Create My SMTP Credentials**.
+    2. Copy the ``Server Name``, ``Port``, ``SMTP Username``, and ``SMTP Password`` values. You'll need these values to configure Mattermost.
+    3. From the ``Domains`` menu, set up and verify a new domain, then enable ``Generate DKIM Settings`` for the domain. We recommend you set up `Sender Policy Framework <https://en.wikipedia.org/wiki/Sender_Policy_Framework>`__ (SPF) and/or `Domain Keys Identified Mail <https://en.wikipedia.org/wiki/DomainKeys_Identified_Mail>`__ (DKIM) for your email domain.
 
-        - `Amazon Simple Email Service (SES) <https://aws.amazon.com/ses/>`_:
-
-            1. Go to `Amazon SES console <https://console.aws.amazon.com/ses>`__ then
-                ``SMTP Settings > Create My SMTP Credentials``
-            2. Copy the ``Server Name``, ``Port``, ``SMTP Username``, and
-                ``SMTP Password`` for Step 2 below.
-            3. From the ``Domains`` menu set up and verify a new domain,
-                then enable ``Generate DKIM Settings`` for the domain.
-
-                1. We recommend you set up `Sender Policy
-                Framework <https://en.wikipedia.org/wiki/Sender_Policy_Framework>`__
-                (SPF) and/or `Domain Keys Identified
-                Mail <https://en.wikipedia.org/wiki/DomainKeys_Identified_Mail>`__
-                (DKIM) for your email domain.
-
-            4. Choose a sender address like ``mattermost@example.com`` and
-                select ``Send a Test Email`` to verify setup is working
-                correctly.
+    4. Choose a sender address like ``mattermost@domain.com`` and select ``Send a Test Email`` to verify setup is working correctly.
 	    
-        - By local ``sendmail`` can be used by using **Server Name** ``127.0.0.1`` with **Port** ``25``
-	- Or by using one of the services listed below in the **Sample SMTP Settings**
+    Alternatively, you can use one of the `services listed below <#sample-smtp-settings>`__, or can set up local ``sendmail`` by setting **Server Name** ``127.0.0.1`` with **Port** ``25``.
 
-2. **Configure SMTP settings**
+    If deploying Mattermost using :doc:`Docker </install/install-docker>`, the standard docker ``172.16.0.0/12`` IP range isn't used. Specify the IP range ``192.168.0.0/24`` to the email service to avoid relay access errors.
```

Comment:
It's not super clear that this refers to the postfix configuration below which ideally should only include the expected network where Mattermost is running on rather than multiple wildcards. 

In the case of our docker setup, `192.168.0.0/24` should work fine, but it could vary depending on how Mattermost is deployed.

Reply:
Thanks, @streamer45! So the postfix and the Docker notes are related, correct? I myself wasn't clear on that based on the issue logged.

Reply:
Correct @cwarnermm , the security exception on the network class is about allowing Mattermost to communicate with postfix so it relates to the latter's config.